spec/forms/normalizes_form_spec.rb

```diff
@@ -19,20 +25,24 @@
       end
     end
 
-    context 'when normalizing attributes with apply_on_nil: true' do
-      it 'applies normalization to nil values' do
+    context 'when normalizing attributes with apply_to_nil: true' do
+      it 'applies normalization to nil values and saves attributes' do
+        expect(nil).to receive(:strip).twice
```

Comment:
It could be more robust and easy to understand with 

`normalizes :first_name, :last_name, with: ->(value) { value.strip || 'Anonymous' }, apply_to_nil: true`

and check 

`expect(user.first_name).to eq('Anonymous')`

Reply:
You're right! Implemented it.